ExamenPGG.Business/Game/Game.cs

```diff
@@ -82,6 +82,7 @@ public void InitializeNewGame(List<IPlayer> playerList)
 
         public async Task StartGame()
         {
+            GameIsBusy = true;
             //some start logic?
```

Comment:
This fixed the issue when coming out of the end view, and starting a new game. i no longer have a crash?